gen3/bin/iam-serviceaccount.sh

```diff
@@ -115,16 +115,16 @@ EOF
 # @return the resulting json from awscli
 ##
 function create_role(){
-  local role_name="${vpc_name}-${SERVICE_ACCOUNT_NAME}-role"
+  local role_name="${1}"
```

Comment:
the role name was already an argument, it just wasn't used: https://github.com/uc-cdis/cloud-automation/blob/251283c6951124730821b6650508de5945c1e259/gen3/bin/iam-serviceaccount.sh#L205